config/packages/security.yaml

```diff
@@ -1,7 +1,7 @@
 security:
     encoders:
         bcrypt:
-            algorithm: bcrypt
+            algorithm: auto
```

Comment:
Why it is needed this change?

Reply:
Bcrypt will be deprecated (gives a warning) - https://symfony.com/blog/new-in-symfony-4-3-native-password-encoder

Reply:
I don't really like that. For everything related with security be explicit is very important, it help to prevent issues and debug easier. I'm ok for migrate to another algo but no to have it auto ;)

Reply:
@jorge07 I've reverted this change, and I'll fix it on another PR because it will affect some changes on src/Domain/User/ValueObject/Auth/HashedPassword.php